package.json

```diff
@@ -53,11 +48,13 @@
     "typescript": "3.1.6"
   },
   "dependencies": {
+    "@types/graphql": "14.0.3",
```

Comment:
I would move it back into `devDeps` since we ship js

Reply:
pls let me do this fix, cause this is my fault.